src/app/components/PortraitVideoCarousel/index.test.tsx

```diff
@@ -15,4 +15,26 @@ describe('PortraitVideoCarousel', () => {
 
     expect(portraitVideoItems.length).toBe(fixture.items.length);
   });
+
+  it('Should render the carousel heading with the correct title', async () => {
+    await act(async () => {
```

Comment:
More out of curiosity - why do we have to wrap the `render` function in an `await act`?